test/functional/rpc_fundrawtransaction.py

```
@@ -938,6 +940,115 @@ def test_subtract_fee_with_presets(self):
+        # the BnB range when these outputs are used.  However if these outputs
+        # are selected, the transaction will end up being too large, so it
+        # shouldn't use BnB and instead fall back to Knapsack but that behavior
+        # is not implemented yet. For now we just check that we get an error.
+        for _ in range(1500):
+            outputs[recipient.getnewaddress()] = 0.1
+        wallet.sendmany("", outputs)
+        self.generate(self.nodes[0], 10)
+        assert_raises_rpc_error(-4, "Transaction too large", recipient.fundrawtransaction, rawtx)
+        self.nodes[0].unloadwallet("large")
+
+    def test_external_inputs(self):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_external_inputs`

Add return type annotation: `None`

(ANN201)